eclipsecs-sevntu-plugin-feature/feature.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<feature id="com.github.sevntu.checkstyle.checks.feature" label="Extension for eclipse-cs plugin with additional Checks" version="1.44.1">
+<feature
+      id="com.github.sevntu.checkstyle.checks.feature"
+      label="Extension for eclipse-cs plugin with additional Checks"
+      version="1.44.1">
```

Comment:
I can only assume the other changes are valid since I don't know, but I noticed when changing the formatting of these line were undone during a release by the release process. I don't know why they were, but I assume they have to remain as they were before unless we can disable the process that re-modified this file.

See https://github.com/sevntu-checkstyle/sevntu.checkstyle/pull/960
The release command that did this is `mvn -e --no-transfer-progress clean install -f eclipse-pom.xml -Plocal-deploy`.